thirdparty/api_models.go

```diff
@@ -50,10 +50,6 @@ type FileDecodeError struct {
 	Message string
 }
 
-func (f FileDecodeError) Error() string {
```

Comment:
I suspect this is necessary, because it implicitly implements the err interface.